src/main/kotlin/pt/up/fe/ni/website/backend/controller/AuthController.kt

```diff
@@ -28,6 +36,21 @@ class AuthController(val authService: AuthService) {
         return mapOf("access_token" to accessToken)
     }
 
+    @PostMapping("/password/recovery")
+    fun generateRecoveryToken(@RequestBody recoveryRequestDto: PasswordRecoveryRequestDto): Map<String, String> {
+        val recoveryToken = authService.generateRecoveryToken(recoveryRequestDto.email)
+            ?: return emptyMap()
+        // TODO: Change to email service
+        return mapOf("recovery_url" to "$recoverPasswordPage/$recoveryToken/confirm")
```

Comment:
Make this return OK all the time and comment on what is supposed to go on an email. If someone wants to test this right away, one can simply print the result. It is safer to leave the correct behavior already implemented.

```suggestion
        val recoveryToken = authService.generateRecoveryToken(recoveryRequestDto.email)
            ?: return emptyMap()
        // TODO:  Add email service with payload "$recoverPasswordPage/$recoveryToken/
        
        return ""
```

Reply:
I'm not sure about the `return ""`, but it should always be status 200. 

Also document on why this is done for security reasons, in order to not leak what are the emails associated with our service

Reply:
Couldn't we have it return the recovery link for now so we can test the feature and remove it when the email service is complete?

If I'm not wrong it always returns status 200.